### ISSUE TITLE
Application unresponsive after resizing the viewport

### ISSUE THREAD

**Issue by roccivic** [pi-hole/AdminLTE#1270]
**In raising this issue, I confirm the following:** `{please fill the checkboxes, e.g: [X]}`

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/AdminLTE/blob/master/CONTRIBUTING.md).
- [x] The issue I am reporting can be *replicated*.
- [x] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/AdminLTE/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/AdminLTE/issues)).

**How familiar are you with the the source code relevant to this issue?:**

`2`

---

**Steps to reproduce:**

* Make sure you're using the boxed layout
* Continuously resize your browser for several seconds

**Expected behaviour:**

* The resize operation completes in a timely fashion

**Actual behaviour:**

* The user interface is unresponsive

**Debug token provided by [uploading `pihole -d` log](https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#debug):**

`not needed`

**Troubleshooting undertaken, and/or other relevant information:**

About 50 seconds of screen capture:
![resize-bug](https://user-images.githubusercontent.com/1194964/81672271-d37a1580-9441-11ea-8983-f57b3f2e9f24.gif)

The performance profile corresponding to the above screen capture (note that each frame takes twice as long to render as the previous):
<img width="635" alt="resize-bug" src="https://user-images.githubusercontent.com/1194964/81672315-e2f95e80-9441-11ea-9e66-e2c810df62d9.png">

The bug is cause by a method named ```_fix``` setting an event listener to itself. Every time the browser fires the ```resize``` event, this method is executed and the number of event listeners associated with the resize event is doubled. This causes an exponential increase in CPU time required to handle the resize event and ultimately bombs the browser tab 💣 bye bye 👋

The bug was originally introduced here:
https://github.com/pi-hole/AdminLTE/commit/0fc943905069672bcc77c3301e085da6a083ba80#diff-a31fe31cbbe6a0f5da4bb6b92f5518a4R422

And now exists in minified source here: https://github.com/pi-hole/AdminLTE/blob/master/scripts/vendor/app.min.js#L13

The code that causes the issue, as it exists today, unminified:
```js
    var e = $(".control-sidebar-bg");
    a._fix(e);
    // ------->%-------
    _fix: function (a) {
        var b = this;
        if ($("body").hasClass("layout-boxed")) {
            a.css("position", "absolute");
            a.height($(".wrapper").height());
            $(window).resize(function () {
                b._fix(a);
            })
        } else {
            a.css({
                position: "fixed",
                height: "auto"
            })
        }
    }
```

The purpose of the code is to adjust the height of the background element in the sidebar to match that of a wrapper.

From the perspective of this bug, the most important part of that snippet is on the first line. Everything that follows operates on the ```control-sidebar-bg``` element.

At the point in time where the bug was introduces, there were 29 html files referencing
the ```control-sidebar-bg``` element:
```sh
$ git checkout 0fc943905069672bcc77c3301e085da6a083ba80 > /dev/null
HEAD is now at 0fc94390 Updated light skin logos
$ git grep -ol control-sidebar-bg | grep html | wc -l
      29
```

Now there are 0:
```sh
$ git checkout devel > /dev/null
Switched to branch 'devel'
$ git grep -ol control-sidebar-bg | grep html | wc -l
       0
```

### In conclusion

1. The entire code snippet pasted above must be removed from the code base, as it references a non-existing element and introduces a bug as per above.

2. There is further dead code referencing the ```control-sidebar-bg``` element a css file, but that's just bloat, doesn't break anything and can be left alone.

If the offending code is removed from the code base, there are no visible side effects. The user interface no longer become unresponsive following a resize operation and each frame now takes about the same amount of time to be rendered:

<img width="1062" alt="Screenshot 2020-05-12 at 11 09 35" src="https://user-images.githubusercontent.com/1194964/81672702-703cb300-9442-11ea-9895-1a6fae504585.png">

**Comment by yubiuser** [comment]
Can confirm the unresponsive behavior.

**Comment by PromoFaux** [comment]
Great Explanation, though see my comments on your #1271 for my concerns in making these changes directly here.

**Comment by XhmikosR** [comment]
Congrats @roccivic for digging in! This is an issue I have been hitting myself since I first installed Pi-hole.

**Comment by roccivic** [comment]
Fixed in #1272
